.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We probably want to add this directory to the `.gitignore` file

Reply:
Done.